fiftyone/utils/paddleocr_vl.py

```
@@ -0,0 +1,233 @@
+    Args:
+        config: a :class:`PaddleOCRVLModelConfig`
+    """
+
+    def __init__(self, config):
+        self._processor = None
+        super().__init__(config)
+
+    def _download_model(self, config):
+        pass  # transformers downloads the model on first load
+
+    def _load_model(self, config):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_load_model`

(ANN202)